web/admin/src/AllAttendees.js

```diff
@@ -44,6 +44,7 @@ export default class AllAttendees extends PureComponent {
   }
 
   render() {
+    this.downloadUserData(this.state.id)
```

Comment:
Should we do this in a componentwill...?

Reply:
If you want to run this before the component is mounted, this can be done in the constructor.  If you want it after the component is mounted, put it in `componentDidMount()`

Reply:
I used componentwillupdate in the new version because it needs to be after component did mount but also if any data changes indirectly